### PR TITLE
crash report issue workflow

### PR DESCRIPTION
**Pull request by Juan-M-V**
# Crash report issue workflow

## Description

Add an extra step to the fuzzer workflow to create an issue if a crash was found

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [ ] CHANGELOG has been updated.

### CODE REVIEW

File: .github/workflows/fuzzer.yml [hunk 1]
```diff
@@ -52,4 +52,25 @@ jobs:
...
+    # Checkout current and last commit for the diff
+    - name: Checkout commits  
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+
+    # Check if any new crashes were added
+    - name: Check diff
+      id: check_file_changed
+      run:  echo "report=$(git diff --name-only HEAD^ HEAD -- test_fuzzer/hfuzz_workspace/test_fuzzer/*.fuzz)" >> $GITHUB_OUTPUT
```
Comment (daphneherlambda) [inline .github/workflows/fuzzer.yml:66]: change here test_fuzzer fir the name of the current workflow fuzzer (fuzz_json)

**Review by daphneherlambda — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/crash_report.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: WARNING crash found
```
Comment (MegaRedHand) [inline .github/ISSUE_TEMPLATE/crash_report.md:2]: We should tag it as `bug`. Also, maybe add some tag specially for this.

**Review by MegaRedHand — APPROVED** [review]

File: .github/workflows/fuzzer.yml (hunk 1 above)
Comment (Juan-M-V) [reply .github/workflows/fuzzer.yml]: Resolved: https://github.com/lambdaclass/cairo-vm/pull/1331/commits/e693103f603315b90fa0e70e3074e92602f164fd

File: .github/ISSUE_TEMPLATE/crash_report.md (hunk 2 above)
Comment (Juan-M-V) [reply .github/ISSUE_TEMPLATE/crash_report.md:2]: Resolved: https://github.com/lambdaclass/cairo-vm/pull/1331/commits/0720d307c39cec2c8638fcaede84009d452100cf

**Review by daphneherlambda — APPROVED** [review]

**Comment by MegaRedHand** [comment]
Ignore the test job failure. It was fixed by #1315

**Review by Oppen — APPROVED** [review]

**Pull request merged**